src/Websites/Websites/ChangeLog.md

```diff
@@ -18,6 +18,7 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Intorduced an option to give custom timeout for `Publish-AzWebApp` 
```

Comment:
Fix typo "Introduced" is there a GitHub Issue tracking this If so please add the GitHub issue # as well for tracking.